python/lsst/ts/standardscripts/base_focus_sweep.py

```diff
@@ -183,6 +196,15 @@ async def configure(self, config: types.SimpleNamespace) -> None:
                         f"{self.camera.components_attr}. Ignoring."
                     )
 
+        if hasattr(config, "focus_steps"):
+            config.focus_window = config.focus_steps[-1] - config.focus_steps[0]
+            config.n_steps = len(config.focus_steps)
```

Comment:
If you update the name above, we will have to assign `config.focus_steps` here.

Reply:
I've decided to use ``focus_step_sequence`` all along so I changed every occurrence of ``focus_steps``. I hope that's ok.